layers/meta-resin-allwinner/conf/machine/orange-pi-one.conf

```diff
@@ -0,0 +1,9 @@
+#@TYPE: Machine
+#@NAME: Orange Pi One
+#@DESCRIPTION: Machine configuration for the Orange Pi One, based on allwinner H3 CPU http://www.orangepi.org/
+
+require conf/machine/include/sun8i.inc
+
+KERNEL_DEVICETREE = "sun8i-h3-orangepi-one.dtb"
+UBOOT_MACHINE = "orangepi_one_defconfig"
+PREFERRED_VERSION_linux-mainline = "4.17.3"
```

Comment:
Why do you declare this conf file here? It already exists in the BSP: https://github.com/linux-sunxi/meta-sunxi/blob/sumo/conf/machine/orange-pi-one.conf

Reply:
I didn't notice it was there, I removed my conf file